### ISSUE TITLE
CMake_VS2015.bat should generate a project for editor .ts sources

### ISSUE THREAD

**Issue by mattbenic** [AtomicGameEngine/AtomicGameEngine#626]
VS2015 supports typescript in web projects, it would be good to have the CMake project generate one to be able to work on the typescript directly in the Atomic solution

**Comment by JoshEngebretson** [comment]
This would be really interesting, we could also create VS2015/VSCode TS projects from the editor (like what Unity does with MonoDevelop).  

There's working being done now to make TS a first class language for projects.  So, there will be better footing on this soon.  We may still have to debug scripts using the editor (at least initially) once the debugger is hooked up...

**Comment by mattbenic** [comment]
Yeah debugging through the editor would be fine, probably preferable to even trying to support the wide array of IDEs you'd need to cover otherwise. Besides which, with all the upgrades coming through to the editor's code editor, you'd probably find a lot of users will just stick to that for everything.
